requirements.txt

```diff
@@ -31,5 +31,6 @@ psqlgraph==1.2.3
 -e git+https://git@github.com/NCI-GDC/cdisutils.git@f54e393c89939b2200dfae45c6235cbe2bae1206#egg=cdisutils
 gdcdictionary==1.2.0
 gdcdatamodel==1.3.12
--e git+https://git@github.com/uc-cdis/indexclient.git@1.5.7#egg=indexclient
+-e git+https://git@github.com/uc-cdis/indexclient.git@master#egg=indexclient
+-e git+https://git@github.com/NCI-GDC/python-signpostclient.git@ca686f55772e9a7f839b4506090e7d2bb0de5f15#egg=signpostclient
```

Comment:
what is the signpostclient dependency for?

Reply:
I think this is some weirdness from updating the branch—this can be removed